### DISCUSSION TITLE
Add LANCache for DediConnect

### DISCUSSION THREAD

**Discussion by mvaessen** [gameserverapp/Platform#433] [Suggestions]
LANCache acts as a caching proxy that helps avoid excess bandwidth and faster downloading by merging calls

https://lancache.net/docs/installation/docker/

**Comment by dagbs** [comment]
I highly recommend setting `CACHE_MEM_SIZE` and `CACHE_DISK_SIZE`. While memory can be managed by the machine (host/docker), disk size will continue to grow until the default (1TB) is hit of which older cached files will start to be removed. If your system does not have 1TB it will continue to fill and eventually stop responding. Minimum size should be at least the total of: game server files + mods + updates for both the game server and its mods (for at least one set of updates).

https://lancache.net/docs/containers/monolithic/#tweaking-cache-sizes

**Comment by dagbs** [comment]
> As of [2020/1/14] Steam Client update, there is now full support in Steam for a lancache. The way it works is by setting a DNS override for `lancache.steamcontent.com` pointing to the IP address of your cache server.

https://lancache.net/news/2020/01/14/steam-client-now-supports-lancache/
